src/apps/backend/modules/config/internals/config.py

```diff
@@ -0,0 +1,156 @@
+import os
+from pathlib import Path
+from typing import Any, Optional
+import yaml
+
+
+class Config:
+    _config_dict: dict[str, Any]
+    _config_path: Path 
+
+    @staticmethod
+    def load_config() -> None:
+        Config._intialize_config()
+        Config._process_custom_environment_variables()
+        print(yaml.dump(Config._config_dict))
+        
+    @staticmethod
+    def get(key: str, default: Optional[Any] = None) -> Any:
+        keys = key.split(".")
+        value = Config._config_dict
+        try:
+            for k in keys:
+                value = value[k]
+            return value
+        except (KeyError, TypeError):
+            return default
+
+    @staticmethod
+    def has(key: str) -> bool:
+        keys = key.split(".")
+        value = Config._config_dict
+        try:
+            for k in keys:
+                value = value[k]
+            return True
+        except (KeyError, TypeError):
+            return False
+    
+    @staticmethod
+    def _read(filename: str) -> dict[str, Any]:
+        file_path = Config._config_path / filename
+        yaml_content: dict[str, Any] = {}
+        try:
+            with open(file_path, "r", encoding="utf-8") as file:
+                yaml_content = yaml.safe_load(file)
+        except FileNotFoundError as e:
+            print(e)
+        return yaml_content
+
+    @staticmethod
+    def _intialize_config() -> None:
+        Config._config_path = Config._get_base_directory(__file__) / "config"
+        default_content = Config._read("default.yml")
+        app_env = os.environ.get("APP_ENV", "development")
+        app_env_content = Config._read(f"{app_env}.yml")
+        merge_content = Config._deep_merge(default_content, app_env_content)
+        Config._config_dict = merge_content
+
+    @staticmethod
+    def _parse_value(value:Optional[str], value_format:str)-> Any:
```

Comment:
Nit: add space after `:`

Kindly scan your code to ensure we are using spacing properly.